src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC, useCallback, useMemo } from 'react';
+import classNames from 'classnames';
+import { Status } from '../../enum/Status';
+import { Todo } from '../../types/Todo';
+
+interface FooterProps {
+  todos: Todo[],
+  onStatusSelect: (status: Status) => void;
+  todoStatus: string;
+}
+
+export const Footer: FC<FooterProps> = ({
+  todos,
+  onStatusSelect,
+  todoStatus,
+}) => {
+  const handleClick = useCallback((
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
```

Comment:
you can pass needed type here like:
```suggestion
    filter: FilterType
```